firefox-ios/Client/Frontend/Reader/View/ReaderModeFontSizeButton.swift

```diff
@@ -19,18 +20,29 @@ class ReaderModeFontSizeButton: UIButton {
 
         switch fontSizeAction {
         case .smaller:
-            setTitle(.ReaderModeStyleSmallerLabel, for: [])
+            configuration?.title = .ReaderModeStyleSmallerLabel
             accessibilityLabel = .ReaderModeStyleSmallerAccessibilityLabel
+            accessibilityIdentifier = AccessibilityIdentifiers.ReaderMode.smallerFontSizeButton
         case .bigger:
-            setTitle(.ReaderModeStyleLargerLabel, for: [])
+            configuration?.title = .ReaderModeStyleLargerLabel
             accessibilityLabel = .ReaderModeStyleLargerAccessibilityLabel
+            accessibilityIdentifier = AccessibilityIdentifiers.ReaderMode.biggerFontSizeButton
         case .reset:
+            configuration?.title = .ReaderModeStyleFontSize
             accessibilityLabel = .ReaderModeResetFontSizeAccessibilityLabel
+            accessibilityIdentifier = AccessibilityIdentifiers.ReaderMode.resetFontSizeButton
         }
 
-        titleLabel?.font = UIFont(
-            name: "SF-Pro-Text-Regular",
-            size: LegacyDynamicFontHelper.defaultHelper.ReaderBigFontSize
-        )
+        guard fontSizeAction != .reset else { return }
+
+        sansSerifFont = FXFontStyles.Regular.title3.scaledFont()
+        serifFont = UIFont(name: "NewYorkMedium-Regular",
```

Comment:
We have `"NewYorkMedium-Regular"` used in two places, is there a way to have that defined somewhere? I tend to prefer having strings that are reused as some constant. What do you think?